graphistry/PlotterBase.py

```diff
@@ -224,7 +224,7 @@ def __repr__(self):
         else:
             return str(rep)
 
-    def addStyle(self, fg=None, bg=None, page=None, logo=None):
+    def add_style(self, fg=None, bg=None, page=None, logo=None):
```

Comment:
@tanmoyio did we flag this as a breaking change in changelog?

Reply:
and instead of breaking people's code, can we add the new method here, and still take the old one, and instead print a deprecation warning?